platform.py

```
@@ -19,234 +19,477 @@
 import subprocess
 import sys
 import shutil
+import logging
+from functools import lru_cache
+from typing import Optional, Dict, List, Any
 from os.path import join
```

Ruff (0.11.9)

`os.path.join` imported but unused

Remove unused import: `os.path.join`

(F401)